### ISSUE TITLE
TF2 / original reblog icon restore extension

### ISSUE THREAD

**Issue by DioVento** [AprilSylph/XKit-Rewritten#1292]
### Concept

![image](https://github.com/AprilSylph/XKit-Rewritten/assets/32431626/a10c0b65-5bb0-40bc-afd8-795f04d099de)
A recent update replaced the little ![image](https://github.com/AprilSylph/XKit-Rewritten/assets/32431626/b2c66531-8513-4559-b2a8-75a13d8cd23b) icon with the word "reblogged". An option to toggle this back would be nice! It also reminded me of ye olde xkit, which had an extension that replaced the ![image](https://github.com/AprilSylph/XKit-Rewritten/assets/32431626/74e7eab2-a04a-44bf-803d-2b9df24457da) with [random killfeed graphics from Team Fortress 2](https://github.com/AprilSylph/XKit-Rewritten/assets/32431626/a5574206-3585-4873-9669-c268f44c79c4) like this: 
![e33](https://github.com/AprilSylph/XKit-Rewritten/assets/32431626/9c4e73e8-4155-4a7c-893b-f032305bad0c)
and it was honestly very cute and fun. If it's not too much trouble, I'd love to see the return of this goofy version, especially if you're already replacing the word "reblogged" with the icon again.

**Comment by AprilSylph** [comment]
> especially if you're already replacing the word "reblogged" with the icon again.

We're not doing that.

**Comment by marcustyphoon** [comment]
Worded slightly confusingly, but there are two requests here:

- XKit script to put back the reblog icon
- Bring back TF2 reblogs

April, are you declining to do the first one from the XKit side or talking about the Tumblr side? I'm not clear on this.

In any case:

- The redesign that includes the word "reblogged" instead of an icon is currently being tested for only a subset of blogs; we won't be doing anything related to the redesign while only some people have it
- I don't think XKit Rewritten generally plans to make and maintain the "goofy" stuff? It would be a pretty simple user script to make, though!

**Comment by AprilSylph** [comment]
> April, are you declining to do the first one from the XKit side or talking about the Tumblr side? I'm not clear on this.

I am declining to do it from the XKit side. My rule for whether to build and/or review an XKit feature/option is whether it's useful or not, while the difference between the reblog icon and some text saying "reblogged" is totally subjective.

Tumblr Fortress 2 holds a special place in my heart too (I can't remember if I built it, but I did write that announcement post), but I just wouldn't be motivated to review a rewritten version. Which sucks, but I haven't figured out how to perform the infinite cognitive energy glitch yet.
